src/fermilib/utils/_operator_utils.py

```diff
@@ -153,8 +153,8 @@ def save_operator(operator, file_name=None, data_directory=None):
 
     tm = operator.terms
     with open(file_path, 'wb') as f:
-        marshal.dump((operator_type, dict(izip(tm.iterkeys(),
-                                          imap(complex, tm.itervalues())))), f)
+        marshal.dump((operator_type, dict(zip(tm.keys(),
```

Comment:
For the dictionary parts you should still be able to use iterkeys and itervalues from future.utils! i.e. `iterkeys(tm)`.